sunpy/database/tests/test_tables.py

```diff
@@ -147,7 +152,7 @@ def test_entries_from_fido_search_result(fido_search_result):
         fileid='EVE_L1_esp_2012001_00',
         observation_time_start=datetime(2012, 1, 1, 0, 0),
         observation_time_end=datetime(2012, 1, 2, 0, 0),
-        size=-1.0,
+        size=None,
```

Comment:
Why has this changed? If it's intentional, it should get a breaking changelog entry

Reply:
added